src/integration/common/SEO.js

```diff
@@ -1,151 +1,184 @@
+      const metaTagContent = getMetaTagContent('meta[property="og:type"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('OG url', () => {
+      const metaTagContent = getMetaTagContent('meta[property="og:url"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('OG site name', () => {
+      const metaTagContent = getMetaTagContent('meta[property="og:site_name"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter card', () => {
+      const metaTagContent = getMetaTagContent('meta[name="twitter:card"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter creator', () => {
+      const metaTagContent = getMetaTagContent('meta[name="twitter:creator"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter image alt', () => {
+      const metaTagContent = getMetaTagContent(
+        'meta[name="twitter:image:alt"]',
+      );
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter image src', () => {
+      const metaTagContent = getMetaTagContent(
+        'meta[name="twitter:image:src"]',
+      );
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter site', () => {
+      const metaTagContent = getMetaTagContent('meta[name="twitter:site"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('OG description', () => {
+      const metaTagContent = getMetaTagContent(
+        'meta[property="og:description"]',
+      );
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('OG title', () => {
+      const metaTagContent = getMetaTagContent('meta[property="og:title"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter description', () => {
+      const metaTagContent = getMetaTagContent(
+        'meta[name="twitter:description"]',
+      );
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Twitter title', () => {
+      const metaTagContent = getMetaTagContent('meta[name="twitter:title"]');
+
+      expect(metaTagContent).toBeTruthy();
+      expect(metaTagContent).toMatchSnapshot();
+    });
+
+    it('Linked data', () => {
+      const linkedDataEl = document.querySelector(
+        'script[type="application/ld+json"]',
+      );
+
+      expect(linkedDataEl).toBeInTheDocument();
+      expect(linkedDataEl.textContent).toBeTruthy();
+      expect(linkedDataEl.textContent).toMatchSnapshot();
```

Comment:
Can we use JSON.parse for the text content, so that it looks nice in the snapshot e.g. 
```
expect(JSON.parse(linkedDataEl.textContent)).toMatchSnapshot();
```

Reply:
Nice suggestion